server/text_generation_server/layers/moe/unquantized.py

```diff
@@ -146,3 +159,110 @@ def _load_expert_weights_row(
     assert all_weight is not None
 
     return all_weight
+
+
+def fused_moe(
```

Comment:
I'd like to keep `moe` on the kernel hub as close to vLLM as possible, so moved this with our own extensions here.